modules/nextflow/src/main/groovy/nextflow/fusion/FusionConfig.groovy

```diff
@@ -37,6 +41,9 @@ class FusionConfig {
 
     final static public String FUSION_PATH = '/usr/bin/fusion'
 
+    final static private Pattern VERSION_JSON = Pattern.compile(/.*\/v(\d+(?:\.\w+)*)-(\w*)\.json$/)
+    final static private Pattern VERSION_TARGZ = Pattern.compile(/.*\/pkg\/(\d+(?:\/\w+)+)\/fusion-(\w+)\.tar\.gz$/)
```

Comment:
```suggestion
    final static private Pattern VERSION_JSON = ~/.*\/v(\d+(?:\.\w+)*)-(\w*)\.json$/
    final static private Pattern VERSION_TARGZ = ~/.*\/pkg\/(\d+(?:\/\w+)+)\/fusion-(\w+)\.tar\.gz$/
```

This can be simplified ☝️